src/config/dynamic-theme-fixes.config

```diff
@@ -5224,6 +5224,9 @@ div[role="button"][aria-disabled="true"] > div {
     --dt-secondary-container: ${#f1f3f4} !important;
 }
 
+button[guidedhelpid="new_menu_button"]{background-color: ${#dbdde0} !important;}
+div[data-target="autopurgebanner"]>section {background: ${#dbdde0} !important;}
```

Comment:
```suggestion
div[data-target="autopurgebanner"] > section {
    background: ${#dbdde0} !important;
}
```

Reply:
done
anyway i've tried to have the same color of the search bar, but when i set the same color in `${...}` form, in dark mode it have another color, and not the expected color... 
in light mode the color of search bar is `f1f3f4`, in dark mode correctly change to `2f3335`; if i set `f1f3f4` to the banner, in dark mode becomes `101314`